libs/search/src/lib/results-list/results-list.container.component.ts

```diff
@@ -15,6 +23,7 @@ export class ResultsListContainerComponent implements OnInit {
   @Input() layout: ResultsListLayout = ResultsListLayout.CARD
```

Comment:
is it useless then ?

Reply:
Changed to have this as default value.

Reply:
I don't get it. `layout` input is not used anymore is it ?

Reply:
If you want to make an app using a custom layout without the layout switcher you can do
```
      <search-results-list-container layout="CARD"></search-results-list-container>
```
no?

Reply:
But in your code the input is not used anyway or i missed something.
The layout is not binded to the result-layout component but to the state. Your app could dispatch `SetResultLayout` action by its own.

Reply:
https://github.com/geonetwork/geonetwork-ui/pull/66/files#diff-040ddc6cd03fe81fc601cb3edff163da7c940c7a551b5e21c66f25d29fe0d11aR45

Reply:
ok sorry